.github/workflows/build-and-test-windows.yml

```diff
@@ -44,3 +44,9 @@ jobs:
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
       - name: Run Unit tests
         run: make gotest GROUP=${{ matrix.group }}
+  unittest:
```

Comment:
please use "windows-unittests" so I can enable easily in the branch protection, which uses the job name and this will conflict with linux names.